dependency_support/pip_requirements.in

```diff
@@ -13,6 +13,7 @@ pyyaml==6.0.1
 # We build most of z3 ourselves but building python is really complicated. Just
 # use pypi version
 z3-solver==4.14.0.0
+zstandard==0.19.0
```

Comment:
Introducing new python dependencies requires that we patch the internal [copybara](https://github.com/google/copybara) logic to rewrite the deps to target internal version of those dependencies.

I would recommend to separate those python tests too for this PR so that it can be worked on separately.

wdyt?